L1Trigger/TrackFindingTracklet/interface/DTCLink.h

```diff
@@ -0,0 +1,54 @@
+// This class holds a list of stubs that are in a given layer and DCT region
+#ifndef DTCLINK_H
+#define DTCLINK_H
+
+#include "L1TStub.h"
+#include "Stub.h"
+
+using namespace std;
```

Comment:
Please don't `using namespace std` in a header file.

Reply:
fixed by migrating to h/cc files